src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -1010,4 +1007,15 @@ private String getLabelString(Node n) {
         }
     }
 
+    /**
+     * Fixes JENKINS-47779 caused by JENKINS-47713
+     * Not using SortedSet because of PluginWrapper doesn't implements equals and hashCode.
+     * @param list original list, probably an unmodifiableList
+     * @return new copy of the list sorted
+     */
+    private static Iterable<PluginWrapper> getSorted(List<PluginWrapper> list) {
+        final List<PluginWrapper> sorted = new LinkedList<PluginWrapper>(list);
+        Collections.sort(sorted);
+        return sorted;
+    }
```

Comment:
The sorting can be done in a generic method, and the method with all the comments could also obtain the plugins from the plugin manager, which is also something repeated in all the uses.

Reply:
Thanks @andresrc